packages/types/src/definitions/core/ProjectData.type.ts

```diff
@@ -1,7 +1,7 @@
 export type ProjectData = {
   title: string;
   description: string;
-  publicUrl: string;
+  publicUrl: string; //TODO: should this also go?
```

Comment:
Yes please, all the public view stuff goes

Users have the custom fields in the project to replace the feature